deps/db/src/logseq/db/sqlite/common_db.cljs

```diff
@@ -282,7 +282,7 @@
 (defn create-kvs-table!
   "Creates a sqlite table for use with datascript.storage if one doesn't exist"
   [sqlite-db]
-  (.exec sqlite-db "create table if not exists kvs (addr INTEGER primary key, content TEXT)"))
+  (.exec sqlite-db "create table if not exists kvs (addr INTEGER primary key, content TEXT, addresses JSON)"))
```

Comment:
`addresses` is separated from `content` so that it can be used to check whether an `addr` is still used by some kv rows.